seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-common/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/AbstractJdbcIT.java

```diff
@@ -350,37 +351,48 @@ protected void initCatalog() {}
 
     @Test
     public void testCatalog() {
-        if (catalog == null) {
-            return;
-        }
-
-        TablePath sourceTablePath =
-                new TablePath(
-                        jdbcCase.getDatabase(), jdbcCase.getSchema(), jdbcCase.getSourceTable());
-        TablePath targetTablePath =
-                new TablePath(
-                        jdbcCase.getCatalogDatabase(),
-                        jdbcCase.getCatalogSchema(),
-                        jdbcCase.getCatalogTable());
-        boolean createdDb = false;
-
-        if (!catalog.databaseExists(targetTablePath.getDatabaseName())) {
-            catalog.createDatabase(targetTablePath, false);
-            Assertions.assertTrue(catalog.databaseExists(targetTablePath.getDatabaseName()));
-            createdDb = true;
-        }
-
-        CatalogTable catalogTable = catalog.getTable(sourceTablePath);
-        catalog.createTable(targetTablePath, catalogTable, false);
-        Assertions.assertTrue(catalog.tableExists(targetTablePath));
-
-        catalog.dropTable(targetTablePath, false);
-        Assertions.assertFalse(catalog.tableExists(targetTablePath));
-
-        if (createdDb) {
-            catalog.dropDatabase(targetTablePath, false);
-            Assertions.assertFalse(catalog.databaseExists(targetTablePath.getDatabaseName()));
-        }
+        Lists.newArrayList(true, false)
```

Comment:
It is recommended to test create index or not in single test case. This method of mixing it with existing test cases is difficult to maintain.

Also, I do not find any assert logic to make sure the index not created.